flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -6,21 +6,54 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
+- commons-beanutils:commons-beanutils:1.9.3
+- commons-codec:commons-codec:1.10
+- commons-collections:commons-collections:3.2.2
+- commons-io:commons-io:2.4
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- com.amazonaws:aws-java-sdk-core:1.11.271
+- com.amazonaws:aws-java-sdk-dynamodb:1.11.271
+- com.amazonaws:aws-java-sdk-kms:1.11.271
+- com.amazonaws:aws-java-sdk-s3:1.11.271
+- com.amazonaws:jmespath-java:1.11.271
 - com.facebook.presto:presto-hive:0.187
 - com.facebook.presto.hadoop:hadoop-apache2:2.7.3-1
+- com.fasterxml.jackson.core:jackson-annotations:2.10.1
+- com.fasterxml.jackson.core:jackson-core:2.10.1
+- com.fasterxml.jackson.core:jackson-databind:2.10.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.6.7
+- com.fasterxml.woodstox:woodstox-core:5.0.3
 - com.google.guava:guava:21.0
+- org.apache.commons:commons-configuration2:2.1.1
+- org.apache.commons:commons-lang3:3.3.2
+- org.apache.hadoop:hadoop-annotations:3.1.0
+- org.apache.hadoop:hadoop-aws:3.1.0
+- org.apache.hadoop:hadoop-auth:3.1.0
+- org.apache.hadoop:hadoop-common:3.1.0
```

Comment:
ah my bad, I messed up the ordering; the `org.*` entries should be below `joda-time`